README.md

```diff
@@ -1 +1,44 @@
-# streaming-service-tracker
\ No newline at end of file
+## App name
+# Track Stream
+
+## Decription
+# Track Stream's purpose is to keep you notified of all the video streaming services you are subscribed too. The application will give you a visual reminder of how much you're spending on streaming services and when the renewal period starts again. 
+
+
+## Technologies used
+# Simple Grid – Simple, responsive CSS Grid system
```

Comment:
let's add a hyperlink to simple grid: https://thisisdallas.github.io/Simple-Grid/